examples/echo_bot.py

```diff
@@ -10,13 +10,28 @@
 import kik_unofficial.datatypes.xmpp.chatting as chatting
 from kik_unofficial.client import KikClient
 from kik_unofficial.callbacks import KikClientCallback
+from kik_unofficial.configuration import env
 from kik_unofficial.datatypes.xmpp.errors import SignUpError, LoginError
 from kik_unofficial.datatypes.xmpp.roster import FetchRosterResponse, PeersInfoResponse
 from kik_unofficial.datatypes.xmpp.sign_up import RegisterResponse, UsernameUniquenessResponse
 from kik_unofficial.datatypes.xmpp.login import LoginResponse, ConnectionFailedResponse
 
-username = sys.argv[1] if len(sys.argv) > 1 else input("Username: ")
-password = sys.argv[2] if len(sys.argv) > 2 else input('Password: ')
+# /// ENVIRONMENT VARIABLES /// #
+# Create your own `.env` file to store the environment variables if running with Docker.
+# See `.env.example` for an example. You can also just set the environment variables manually.
+BOT_USERNAME = env.get("BOT_USERNAME", None)
+BOT_PASSWORD = env.get("BOT_PASSWORD", None)
+BOT_NODE_JID = env.get("BOT_NODE_JID", None)
+
+
+if BOT_USERNAME == "" or BOT_USERNAME is None:
```

Comment:
Can these variables actually contain `""`? I ask because they seem to get initialized to `None` above.

Reply:
Unfortunately, yes. The python-dotenv dependency acts as follows:

`MY_VAR=` yields `""`
`MY_VAR` yields `None`
Not including `MY_VAR` at all in `.env` yields `None`

I created [`credential_utilities.py`](https://github.com/tomer8007/kik-bot-api-unofficial/pull/224/commits/2bc8394bf95fa62785623e575f9b3e524f782d29) and updated the logic to use falsy checks for brevity instead.